content/lwrp_to_custom_resources.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Migrating from LWRPs to Custom Resources"
+gh_repo = "chef-web-docs"
+
+product = ["client", "workstation"]
+
+[menu]
+  [menu.infra]
+    title = "Migrating from LWRPs"
+    identifier = "chef_infra/resources/custom_resources/lwrp"
+    parent = "chef_infra/resources/custom_resources"
+    weight = 120
++++
+
+## Overview
+
+It is no longer recommended to write resources in the __Light Weight Resource Provider (LWRP)__ format.
+
+This guide describes how to migrate from an existing LWRP to a Custom Resource.
+
+If you are creating a Custom Resource from scratch please see the [Custom Resource Getting Started Guide](getting-started.md) instead.
+
+## Convert files to Custom Resources Layout
+
+LWRPs consist of two library files: a resource and a provider for that resource.
```

Comment:
If there are two library files it's not an LWRP. It's an HWRP. LWRPs have a file in /resources and a file in /providers.

Reply:
This whole page is really about HWRPs. It should just be a guide to converting from HWRPs

Reply:
I'm opening a 'modernization' pr in #3586. We'll fix it there.